x-pack/legacy/plugins/ml/public/application/services/results_service/result_service_rx.ts

```diff
@@ -532,3 +532,12 @@ export function getScheduledEventsByBucket(
       })
     );
 }
+
+export function fetchPartitionFieldsValues(
+  searchTerm: Record<string, string>,
```

Comment:
Nit: See the comment above about Record vs index signature.

Reply:
Fixed in https://github.com/elastic/kibana/pull/53879/commits/c66af6e20807829e3fa8fcbb511d75cbb278d0dd